tools/webpack-core/src/plugins/BanImportsPlugin.ts

```diff
@@ -1,48 +0,0 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
why is this being nuked?

Reply:
Currently it doesn't even compile. And it's not a part of the default plugins, so we decided to remove it.

Reply:
What do you mean i doesn't compile? 

It's not being used directly in the DefaultsPlugin, but it was being used in our cra fork (pre the 5.0.0 version)
I do agree we can probably get rid of this, but removing this altogether would be a breaking change. Should amrk this as deprecated and remove it in a later version. imo we could handle this via lint rules instead

Reply:
Sorry, "doesn't compile" was a bad term. What I meant is that it doesn't work. I tested it by temporarily adding it to the DefaultsPlugin and I got "cannot read property of undefined" or something along those lines. My guess is that at some point there were some breaking changes on that hook and we didn't catch it since we are using `any` pretty much everywhere in that plugin (it's basically a js file, not ts), so it kind of just stayed broken for a while.

Most likely CRA 4.x doesn't import the current package version.